docs/source/asv.conf.json.rst

```diff
@@ -22,6 +22,10 @@ this file and their expected values.
 
 .. only:: not man
 
+``version``
+-----------
+Version of the `asv.conf.json` spec. Currently only `1`.
```

Comment:
Could you add the version instead of currently? i.e. as of version ... since it might change.

Reply:
Sounds good, updated!